### PR TITLE
Enabling JS/Py init packages, and fixing bugs.

### PR DESCRIPTION
**Pull request by Alfred-Mountfield**
## 🌟 What is the purpose of this PR?

Up until now, only the JSON init package has been enabled. This PR conditionally enables the appropriate init package, and fixes associated bugs with the JS/PY init implementation.

## 🔗 Related links

- [Asana task description](https://app.asana.com/0/1199550852792314/1201644875152933/f)

## 🔍 What does this change?

- Manually configures the init package in the `package::ConfigBuilder` according to the init src type provided by the CLI
- Disables calls related to `Error.prepareStackTrace` in the package implementation until that's fixed [in this task](https://app.asana.com/0/1199550852792314/1201637811192668/f)
- Renames the jspy init package module to js_py to fix import paths resultant from serialization of camelcase
- Fixes serialization of the package.js implementation for init JS/Py
- Improves a few confusing error messages

## 🛡 Tests

- ✅ Manual Tests

### CODE REVIEW

**Comment by Alfred-Mountfield** [comment]
Marking as draft until #150 is merged to avoid breaking main

File: packages/engine/src/simulation/package/init/packages/js_py/package.js [hunk 1]
```diff
@@ -88,14 +90,18 @@ const run_task = (experiment, _sim, task_message, _group_state, context) => {
 
     let data;
     try {
-        data = JSON.stringify(agents);
+        data = {
+            "SuccessMessage": {
+                "agent_json": JSON.stringify(agents)
+            }
+        };
```
Comment (litvand) [inline packages/engine/src/simulation/package/init/packages/js_py/package.js:101]: We should revisit this at some point to remove the need for `JSON.stringify` here.

**Review by litvand — APPROVED** [review]

File: packages/engine/src/simulation/package/init/packages/js_py/package.js (hunk 1 above)
Comment (Alfred-Mountfield) [reply packages/engine/src/simulation/package/init/packages/js_py/package.js:101]: When we were working through it I thought the same, just realised an easy solution, have updated the PR

File: packages/engine/src/simulation/package/init/packages/js_py/package.js [hunk 2]
```diff
@@ -82,18 +82,23 @@ const run_task = (experiment, _sim, task_message, _group_state, context) => {
...
   try {
-    data = JSON.stringify(agents);
+    data = {
+            "SuccessMessage": {
+                "agents": agents
+            }
+        };
   } catch (e) {
     throw new Error(
       `could not serialize init return value to JSON: ${e.message}`,
```
Comment (litvand) [inline packages/engine/src/simulation/package/init/packages/js_py/package.js:104]: This error message is incorrect now.

**Review by litvand — CHANGES_REQUESTED** [review]

File: packages/engine/src/simulation/package/init/packages/js_py/package.js (hunk 2 above)
Comment (Alfred-Mountfield) [reply packages/engine/src/simulation/package/init/packages/js_py/package.js:104]: I'm not sure it's incorrect, just shouldn't be able to happen. Will update and remove

**Review by litvand — APPROVED** [review]

**Pull request merged**